rca/settings/base.py

```diff
@@ -785,3 +785,9 @@
     ENQUIRE_TO_STUDY_DESTINATION_EMAILS = env.get(
         "ENQUIRE_TO_STUDY_DESTINATION_EMAILS"
     ).split(",")
+
+
+# Silenced system checks
+SILENCED_SYSTEM_CHECKS = [
+    "templates.E003",  # Warns if two templatetags have the same name
+]
```

Comment:
```suggestion
```

**suggestion** I think it's better to see the warning and at some point we should investigate a fix where this is seen.